src/client/protocol.c

```diff
@@ -502,17 +502,25 @@ int clientSendPrepare(struct client_proto *c, const char *sql, struct client_con
 	struct request_prepare request;
 	request.db_id = c->db_id;
 	request.sql = sql;
-	REQUEST(prepare, PREPARE, 0);
+	REQUEST(prepare, PREPARE, DQLITE_PREPARE_STMT_SCHEMA_V1);
```

Comment:
Maybe something to think about later, but what should happen when a dqlite server doesn't know a certain schema version? Should the client try to downgrade the request? In case of a prepare stmt, the downgrade to `V0` should work in most cases as I expect that most of the SQL that has to be prepared will only contain a single statement.